### ISSUE TITLE
Show identityId of pod identity in kubectl get triggerauthentication -o wide

### ISSUE THREAD

**Issue by tomkerkhove** [kedacore/keda#4527]
### Proposal

Show identityId of pod identity in kubectl get triggerauthentication -o wide to simplify investigation.

It should not specifically be a new field but can be `azure-workload (foo)` and `azure-workload (default)`/`azure-workload` when no identity ID was specified

### Is this a feature you are interested in implementing yourself?

No

**Comment by JorTurFer** [comment]
Currently, that value is generated from .spec:
![image](https://user-images.githubusercontent.com/36899226/236695290-0bae304f-0c66-4b8b-9a05-3d2a71f402e2.png)

But we could introduce an internal status field with the information. The problem here IMHO will be that the real use case isn't `azure-workload (foo)`, it is `azure-workload (c27bf924-db20-4972-a528-b96ecb878895)`. 
I'm not totally sure if this will be useful. we can't provide other info more than the given GUID

**Comment by tomkerkhove** [comment]
> The problem here IMHO will be that the real use case isn't `azure-workload (foo)`, it is `azure-workload (c27bf924-db20-4972-a528-b96ecb878895)`.

That is the same actually, I was just lazy and used `foo` as example for app ID :)

> I'm not totally sure if this will be useful. we can't provide other info more than the given GUID

That is exactly what is needed to be able to see if the App ID was correctly configured without having to open the YAML which I had to do due to https://github.com/kedacore/keda/issues/4528#issuecomment-1537508517

**Comment by JorTurFer** [comment]
Is the GUID useful? I mean, just having 2 states could solve it? My doubts are related with the GUID itself, maybe something like `azure-workload (defalt)` and `azure-workload (provided)` or something similar could be more useful, not having 36 characters in the output for seeing the GUID

**Comment by tomkerkhove** [comment]
Yes it is useful, because then you see what is configured. Provided is not very useful as you still have to check the details while the GUID shows exactly what you need imo

**Comment by JorTurFer** [comment]
I think that we can do it if you believe that it's useful. I'm not sure if getting the ID is useful but it's doable IMO

**Comment by tomkerkhove** [comment]
I think it does because it clearly shows the used ID, but I don't want to push this. @kedacore/keda-maintainers thoughts?

**Comment by tomkerkhove** [comment]
> I think it does because it clearly shows the used ID, but I don't want to push this. @kedacore/keda-maintainers thoughts?

Bump @kedacore/keda-maintainers

**Comment by tomkerkhove** [comment]
@JorTurFer @zroubalik Any thoughts?

**Comment by JorTurFer** [comment]
I think that this can be useful. Maybe we can update the status with the used ID and just expose it as response of the command

**Comment by zroubalik** [comment]
I am okay with that

**Comment by tomkerkhove** [comment]
@SpiritZhou Can you work on this when you have some time?
